resources/options_lookup.tsv

```diff
@@ -1223,389 +1223,389 @@ HVAC System Is Shared	Only Heating Shared
-		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
+		ResidentialAirflow	has_hvac_flue=false																																												
+		ResidentialHVACSizing																																													
 HVAC System Shared Electricity	Fan Coil Cooling Only	ResidentialHVACCentralSystemFanCoil	fan_coil_heating=false	central_boiler_fuel_type=electric																																											
-		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
+		ResidentialAirflow																																													
+		ResidentialHVACSizing																																													
 HVAC System Shared Electricity	Boiler Baseboards Heating Only	ResidentialHVACCentralSystemBoilerBaseboards	"central_boiler_system_type=hot water, forced draft"	central_boiler_fuel_type=electric																																											
-		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
+		ResidentialAirflow	has_hvac_flue=false																																												
+		ResidentialHVACSizing																																													
 HVAC System Shared Electricity	PTAC Heating And Cooling	ResidentialHVACCentralSystemPTAC	central_boiler_fuel_type=electric																																												
-		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
+		ResidentialAirflow	has_hvac_flue=false																																												
+		ResidentialHVACSizing																																													
 HVAC System Shared Natural Gas	None																																														
 HVAC System Shared Natural Gas	Fan Coil Heating And Cooling	ResidentialHVACCentralSystemFanCoil	fan_coil_heating=true	central_boiler_fuel_type=gas																																											
 		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
+		ResidentialHVACSizing																																													
 HVAC System Shared Natural Gas	Fan Coil Cooling Only	ResidentialHVACCentralSystemFanCoil	fan_coil_heating=false	central_boiler_fuel_type=gas																																											
-		ResidentialAirflow	has_hvac_flue=true																																												
-		ResidentialHVACSizing	show_debug_info=false																																												
+		ResidentialAirflow																																													

```

Comment:
I'm totally confused here. 

1) Why is the `has_hvac_flue` argument value being changed for some options? Was it wrong before?
2) Why do some "HVAC System Shared" options have the `has_hvac_flue` argument defined while others have the argument removed? There seems like no consistency?

Reply:
For all the Shared Electricity system options I changed `has_hvac_flue=true` to `has_hvac_flue=false` because these systems would never have a flue.

Then I removed `has_hvac_flue` from all of the Cooling Only shared system options because `has_hvac_flue` would be supplied by the non-shared heating system option.

See https://github.com/NREL/OpenStudio-BuildStock/commit/d2844b2d137123940b172df409ac2747d2e24e1d.

Reply:
I see. Thanks for the explanation.